data/json/items/archery.json

```diff
@@ -2,258 +2,247 @@
   },
   {
     "type": "AMMO",
     "id": "arrow_wood",
     "price": 1000,
-    "name": "wooden arrow",
+    "name": "wooden broadhead arrow",
     "symbol": "=",
     "color": "green",
-    "description": "A basic wooden arrow, it has a metal arrowhead and fletching.  It's lightweight, does some damage, and is so-so on accuracy.  Stands a below average chance of remaining intact once fired.",
-    "material": "wood",
+    "description": "A fletched wooden arrow shaft with a bladed tip.  Useful for maximising damage to the target.  Stands a decent chance of remaining intact once fired.",
+    "material": [ "wood", "steel" ],
     "volume": 1,
-    "weight": 51,
+    "weight": 55,
     "bashing": 2,
     "cutting": 1,
     "ammo_type": "arrow",
-    "damage": 10,
-    "pierce": 3,
-    "range": 7,
-    "dispersion": 135,
+    "prop_damage": 1.25,
+    "pierce": 1,
+    "dispersion": 110,
     "loudness": 0,
     "count": 10,
-    "effects": [ "RECOVER_5" ]
+    "effects": [ "RECOVER_25" ]
   },
   {
     "type": "AMMO",
     "id": "arrow_heavy_fire_hardened_fletched",
-    "price": 1000,
-    "name": "heavy fire-hardened arrow",
+    "price": 300,
+    "name": "simple wooden arrow",
     "symbol": "=",
     "color": "green",
-    "description": "This arrow is significantly sturdier than other wooden arrows.  It has a fire-hardened point and fletching.  Stands a good chance of remaining intact once fired.",
+    "description": "A simple fletched wooden arrow shaft with a pointed tip.  Stands a low chance of remaining intact once fired.",
     "material": "wood",
     "volume": 1,
-    "weight": 49,
+    "weight": 35,
     "bashing": 4,
     "ammo_type": "arrow",
-    "damage": 5,
-    "pierce": 2,
-    "range": 7,
-    "dispersion": 135,
+    "prop_damage": 0.7,
+    "range": -4,
+    "dispersion": 180,
     "loudness": 0,
     "count": 10,
-    "effects": [ "RECOVER_10" ]
+    "effects": [ "RECOVER_6" ]
   },
   {
     "type": "AMMO",
     "id": "arrow_heavy_field_point_fletched",
     "price": 1000,
-    "name": "heavy field point arrow",
+    "name": "wooden small game arrow",
     "symbol": "=",
     "color": "green",
-    "description": "This arrow is significantly sturdier than other wooden arrows.  It has a crude field point head and fletching.  Stands a good chance of remaining intact once fired.",
-    "material": "wood",
+    "description": "A fletched wooden arrow shaft with a blunt tip.  Useful for hunting small woodland creatures without splattering them all over the ground.  Stands a decent chance of remaining intact once fired.",
+    "material": [ "wood", "steel" ],
     "volume": 1,
-    "weight": 57,
+    "weight": 60,
     "bashing": 4,
     "ammo_type": "arrow",
-    "damage": 6,
-    "pierce": 3,
-    "range": 7,
-    "dispersion": 135,
+    "prop_damage": 0.5,
+    "dispersion": 130,
     "loudness": 0,
     "count": 10,
-    "effects": [ "RECOVER_10" ]
+    "effects": [ "RECOVER_35" ]
```

Comment:
If this is a game arrow, does it need NOGIB here?

Reply:
It does, thank you for the save.